packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -220,10 +218,11 @@ class DirectStripeRoutes {
       planId,
       planName: selectedPlan.plan_name,
       productName: selectedPlan.product_name,
-      planEmailIconURL: planMetadata.emailIconURL,
-      planDownloadURL: planMetadata.downloadURL,
-      appStoreLink: planMetadata.appStoreLink,
-      playStoreLink: planMetadata.playStoreLink,
+      planEmailIconURL: productMetadata.emailIconURL,
```

Comment:
Renamed this variable from `planMetadata` to `productMetadata` for consistency with other emails usage of the same data.